src/docs/cli.js

```diff
@@ -0,0 +1,21 @@
+import { execFileSync } from 'child_process';
+import { resolve } from 'path';
+
+const kibanaDir = resolve(__dirname, '..', '..');
+const docsIndexFile = resolve(kibanaDir, 'docs', 'index.asciidoc');
+const docsToolDir = resolve(kibanaDir, '..', 'docs');
```

Comment:
Could we add a fallback for `elastic-docs`? I can't stand having a `docs` project in my dev directory